Add docs for `UV_PROJECT_ENVIRONMENT` (#6987)
astral-sh/uv: docs/concepts/projects.md

    managed = false
    ```

### Custom project environment paths

The `UV_PROJECT_ENVIRONMENT` environment variable can be used to configure the path to virtual

> environments in projects. If a relative path is provided, it will be resolved relative to the

Reviewer: "path to the project vitrual environment"?

Reviewer: Mention the default?

workspace root.

If an environment is not present at the given path, uv will create it.

> This option can be used to write to the system Python environment, though it is not recommended.
> `uv sync` will remove extraneous packages from the environment by default, which means if there are
> installed packages relevant to the operation of the system, the system may be broken.

Reviewer: ```suggestion
This option can be used to write to the system Python environment, though it is not recommended.
`uv sync` will remove extraneous packages from the environment by default and, as such, may
leave the system in a broken state.
```
